workflows/create-pull-request.yaml

```diff
@@ -0,0 +1,59 @@
+name: Create Pull request
+
+on:
+  schedule:
+    # Runs "at minute 44 past midnight" (see https://crontab.guru)
+    - cron: "44 0 * * *"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    env:
+      bot_username: BotName
+      bot_email: BotMail@gmail.com
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ["de-german", "fr-french"]
+    steps:
+      - name: Fetch translation repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: solidity-docs-test/${{ matrix.repos }}
+          fetch-depth: "0"
+
+      - name: Fetch this repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: solidity-docs/translation-guide
+          path: .github-workflow
+
+      - name: Create merge commit
+        id: merge
+        run: |
+          ./.github-workflow/scripts/merge-conflicts.sh
+          ./.github-workflow/scripts/generate-PR-body.sh
+
+      - name: Remove this repository
+        run: |
+          rm -rf .github-workflow
+
+      - name: Create Pull Request
+        id: cpr
```

Comment:
The `id` is not necessary if you're not referencing the action elswhere in the config.

Reply:
done